solr4/source/java/org/alfresco/solr/tracker/ModelTracker.java

```diff
@@ -460,40 +466,39 @@ else if (qName.contains(":"))
      */
     private void removeMatchingModels(File alfrescoModelDir, QName modelName)
     {
-
-        final String prefix = modelName.toPrefixString(this.infoSrv.getNamespaceDAO()).replace(":", ".") + ".";
-        final String postFix = ".xml";
-
-        File[] toDelete = alfrescoModelDir.listFiles(new FileFilter()
-        {
-            @Override
-            public boolean accept(File pathname)
-            {
-                if (pathname.isDirectory()) { return false; }
-                String name = pathname.getName();
-                if (false == name.endsWith(postFix)) { return false; }
-                if (false == name.startsWith(prefix)) { return false; }
-                // check is number between
-                String checksum = name.substring(prefix.length(), name.length() - postFix.length());
-                try
-                {
-                    Long.parseLong(checksum);
-                    return true;
-                }
-                catch (NumberFormatException nfe)
-                {
-                    return false;
-                }
-            }
-        });
-
-        if (toDelete != null)
-        {
-            for (File file : toDelete)
-            {
-                file.delete();
-            }
-        }
+    	final String prefix = modelName.toPrefixString(this.infoSrv.getNamespaceDAO()).replace(":", ".") + ".";

```

Comment:
If I haven't missed anything there is no code change in this block. Not so sure if it was line ending changes.